### PR TITLE
Documentation ready for gh-pages

### PR DESCRIPTION
**Pull request by wyusuf068**
Pkgdown completed, website is built. Need to set up gh-pages

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    person(given = "Warsame",
+           family = "Yusuf",
+           role = c("aut", "cre"),
+           email = "waryusuf@ohri.ca"),
+    person(given = "Rostyslav", 
+           family = "Vyuha",
+           role = c("aut"),
+           email = "rostikvyuha@gmail.com"))
+Description: What the package does (one paragraph).
+License: What license it uses
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply DESCRIPTION]: Yes

File: README.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
 # cchsflow
 
-This repository contains supports the use of the Canadian Community Health Survey (CCHS). The current focus is transformation of harmonized variables across surveys from 2001 to 2014. 
+This package contains supports the use of the Canadian Community Health Survey (CCHS). The current focus is transformation of harmonized variables across surveys from 2001 to 2014. At the heart of `cchsflow` are two worksheets (CSV files) that describe how to transform variables different CCHS cycles into common variables: `variables.csv` and `variableDetails.csv`.
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply README.md]: This README is behind the master branch. Let’s update the readme after we merge. Unless @warsame wants to update the PR with those edits in master.

File: Vignettes/usingcchsflow.Rmd [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+install_github("Big-Life-Lab/bllflow", ref = "recode-with-table-patch")
+```
+```{r results= 'hide', message = FALSE, warning=FALSE}
+library(bllflow)
+```
+
+To be able to combine datasets with minimal issues, we recommend using the `bind_rows` function in the [dplyr](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8) package. 
+
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(dplyr)
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply Vignettes/usingcchsflow.Rmd]: Yes.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (WarSame) [reply README.md]: I think you have the wrong person tagged.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Filing an issue
+
+The easiest way to propose a change or new feature is to file an issue. If you've found a
+bug, you may also create an associated issue. If possible, try to illustrate your proposal or the bug with a minimal [reproducible example](https://www.tidyverse.org/help/#reprex).
+
+## Pull request
+
+*  Please create a Git branch for each pull request (PR).
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply CONTRIBUTING.md]: I think Master may be ahead for this section.

File: Vignettes/usingcchsflow.Rmd [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+  + `varLabels` is set to NULL. This argument can be used to add labels to a transformed dataset that only contains a subset of variables from `variables.csv`. The format to add labels to a subset of variables is `c(variable = "Label")`.
+  
+See the `bllflow::RecWTable()` [documentation](http://bllflow.projectbiglife.ca) for more details about how to use RecWTable().
+
+### Example 1. Transform a single variable from a single database
+
+In this example, the sex variable in the 2001 CCHS cycle is transformed.
+
+```{r, warning=FALSE}
+sex2001 <- RecWTable(dataSource = cchsMock2001, variableDetails = varDetails, datasetName = "cchs-82M0013-E-2001-c1-1-general-file", log = TRUE, variables = c("DHH_SEX"))
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply Vignettes/usingcchsflow.Rmd]: A challenge with the cchsflow and the vignette is that you need quite a bit to get going. The ideal package has a nice one or two-line intro example. No specific suggestions, other than agreeing with Yulric. We need to carefully walk people through if there are a few steps.

File: Vignettes/variableDetails.Rmd [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+library(DT)
+datatable(varDetails[c(75:79), ], options=list(columnDefs = list(list(visible=FALSE, targets=c(4:16)), list(className = 'dt-center', targets = 0:3)), pageLength = 5))
+```
+
+4. **databaseStart:** the CCHS surveys that contain the variable of interest, separated by commas. Each CCHS survey contains a unique identifier in DDI document.
+
+The CCHS database identifier and other infomation can be extracted using `bllflow::ReadDDI()`
+
+```{r, echo=TRUE, warning=FALSE}
+CCHS2001_DDI <- bllflow::ReadDDI(file.path(getwd(), "../inst/extdata/CCHS_DDI"), "cchs-82M0013-E-2001-c1-1-general-file.xml")
```
[the remark this answers is not in the document]
Comment (DougManuel) [reply Vignettes/variableDetails.Rmd]: The PDF names are generating CRAN warnings because they are too long. I think maybe the XML documents as well,

**Pull request merged**